Dockerfile.compile

```diff
@@ -1,11 +1,9 @@
 FROM alpine:3.11
 RUN apk add --update \
     openjdk11 \
-    openjdk8 \
     && rm -rf /var/cache/apk
 LABEL maintainer="sig-platform@spinnaker.io"
 ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk
```

Comment:
I've been removing this `JAVA_HOME` too... there's only one JDK installed so we don't need it and it feels like one less thing to fix if we change distros or something like that...